docs/userGuide/makingTheSiteSearchable.md

```diff
@@ -26,6 +26,13 @@
 <include src="syntax/searchBars.mbdf" />
 <include src="syntax/keywords.mbdf" />
```

Comment:
This thing is in the wrong place, so the tipboxes below are out-of-position.

Reply:
Sorry, I'm a little confused by this! What is in the wrong place?

Reply:
From the reader's point of view, we are jumping around the searchbar and the keywords.

See Netlify to see what I mean:

![searchbar](https://user-images.githubusercontent.com/3168908/53462009-471e4580-3a7d-11e9-8f9e-7a02a66b1e98.png)

Reply:
An alternative suggestion is to convert the tipboxes into a section of its own, so that it can be de-associated from the search bar (i.e. `## Using External Search Providers`).

Reply:
I had the same thoughts about the tipboxes at the end. Could we have it at the top? Just after the
`All headings of levels 1-3 are captured in the search index by default. You can change this setting using the headingIndexLevel property of the site.json.`

The idea is to redirect the user asap if he was looking to make the site searchable with Algolia.

Reply:
Done!